vanilla-framework/vanilla-framework
# Remove deprecated components

## Done

I've removed all code marked as deprecated for 2.0.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- See that there are no visual changes to components

## Details

Fixes #2223
